src/app/FakeLib/NuGet/NugetHelper.fs

```diff
@@ -91,6 +93,8 @@ let NuGetDefaults() =
       WorkingDir = "./NuGet"
       PublishUrl = null
```

Comment:
Do you think we should change this default or do it like nuget and throw an error? (I know its unrelated to this PR, but it might make sense to fix that as well)